lib/content-services/src/lib/security/services/security-controls-groups-marks-security.service.ts

```diff
@@ -0,0 +1,259 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, from, Observable, Subject } from 'rxjs';
+import { SecurityControlsGroupResponse } from './models/security-controls-group-response.interface';
+import { SecurityControlsMarkResponse } from './models/security-controls-mark-response.interface';
+import {
+    SecurityGroupsApi,
+    SecurityMarksApi,
+    SecurityGroupEntry,
+    SecurityMarkEntry,
+    SecurityGroupBody,
+    SecurityMarkPaging,
+    SecurityMarkBody,
+    SecurityGroupPaging,
+    AuthorityClearanceApi,
+    AuthorityClearanceGroupPaging,
+    NodeSecurityMarkBody
+} from '@alfresco/js-api';
+import { AlfrescoApiService } from '@alfresco/adf-core';
+import { finalize } from 'rxjs/operators';
+
+const DEFAULT_MAX_GROUPS = 10;
```

Comment:
maybe instead of setting new max constant we should to reuse UserPreferencesService.paginationSize?

Reply:
done